app/src/main/java/eu/kanade/tachiyomi/util/view/ControllerExtensions.kt

```diff
@@ -775,6 +776,11 @@ fun Controller.requestFilePermissionsSafe(
             }
             .setNegativeButton(android.R.string.cancel, null)
             .show()
+    } else {
```

Comment:
Since this else case will enter if the user denies the permission, you should add this check too
```suggestion
    } else if Build.VERSION.SDK_INT < Build.VERSION_CODES.R || Environment.isExternalStorageManager() {
```